libs/logging-bundle/azure-pipelines.tests.yml

```diff
@@ -2,15 +2,15 @@ jobs:
   - template: ../../azure-pipelines/jobs/run-tests.yml
     parameters:
       displayName: Tests (Symfony 5.4)
-      serviceName: dev74
+      serviceName: dev
```

Comment:
idealne by to melo mit svoje pro ci, ale k8s client ji taky zatim nema
navic tady nejsou zadny specialni envy